src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -100,14 +109,19 @@ export class SkySelectFieldComponent implements ControlValueAccessor {
   @Input()
   public pickerHeading: string;
 
+  @Output()
+  public blur = new EventEmitter();
```

Comment:
This needs to be completed when the component is destroyed.
```
this.blur.complete();
```

Reply:
done